ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -9,15 +9,15 @@
         data-test-isReindexing />
     </div>
   {{/if}}
-
-  <div class="selectable-card-container">
+  <div
+    class="selectable-card-container {{if isSecondary 'secondary' 'primary'}}"
```

Comment:
so here i think it would be `{{unless (eq secondaryId null) 'secondary' 'primary'}}`

Reply:
Because you _can_ be a Secondary, but not have a `secondaryId`, we can't really do this one.  ☹️